pubspec.yaml

```diff
@@ -1,6 +1,8 @@
 name: expected_output
 version: 1.2.2-dev
-description: Easily write and test multiline input and expected output
+description: >
```

Comment:
Use `>-` to collapse inner newlines, too!

Reply:
Done!